hw/top_earlgrey/syn/asic.constraints.sdc

```diff
@@ -70,12 +70,12 @@ set_clock_uncertainty ${SETUP_CLOCK_UNCERTAINTY} [get_clocks IO_CLK]
 
 # generated clocks (div2/div4)
 create_generated_clock -name IO_DIV2_CLK -divide_by 2 \
-    -source [get_pins top_earlgrey/u_clkmgr_aon/u_io_div2_div/u_clk_mux/gen_*u_impl*/u_size_only_mux2/D1] \
-    [get_pins top_earlgrey/u_clkmgr_aon/u_io_div2_div/u_clk_mux/gen_*u_impl*/u_size_only_inv/${DRIVING_CELL_PIN}]
+    -source [get_pins top_earlgrey/u_clkmgr_aon/u_no_scan_io_div2_div/u_clk_mux/gen_*u_impl*/u_size_only_mux2/D1] \
+    [get_pins top_earlgrey/u_clkmgr_aon/u_no_scan_io_div2_div/u_clk_mux/gen_*u_impl*/u_size_only_inv/${DRIVING_CELL_PIN}]
```

Comment:
These constraints won't work anymore, as they were formulated using the pins of the scan bypass mux cell that is now removed.

Hold on a minute, I need to spin this up in an interactive session to figure out the new constraints.

Reply:
ah good catch, i didn't notice that.
maybe i can add a buffer in place of the mux?

Reply:
@msfschaffner can you have a look at the latest?